tools/js-tools/jest/setup-globals.js

```diff
@@ -1,3 +1,5 @@
+/* global window, jest */
```

Comment:
Odd, `window` isn't already available as a global?

Reply:
In the old config we told eslint that everything was `node` and `browser` and `jquery` (and `es6` and `jest`!).

I decided to trim that back in the new config. In particular `tools/js-tools/eslint.config.mjs` now specifies only `node`, no longer `browser` or `jest`, since most of the stuff under `tools/js-tools/` doesn't run in a browser (or [jsdom](https://www.npmjs.com/package/jsdom)) context.

It's possible to override that in the `eslint.config.mjs` rather than by using `/* global */` (see `makeEnvConfig()`), but since this was the only one in here I somewhat arbitrarily chose `/* global */`.